src/main/flight/mixer.h

```diff
@@ -97,6 +97,8 @@ typedef struct servoParam_t {
     int16_t max;                            // servo max
     int16_t middle;                         // servo middle
     int8_t rate;                            // range [-100;+100] ; can be used to ajust a rate 0-100% and a direction
+    int8_t minLimit;                        // range [-90;90] ; can be used to adjust a rate 0-100% and a direction
+    int8_t maxLimit;                        // range [-90;90] ; can be used to adjust a rate 0-100% and a direction
```

Comment:
to clarify, say the min limit was -45 and the max limit was +45, what would happen if the user got the values for max and max transposed and specfied min +45 and max -45?